src/StaticFAQ/ArticleFeedback/ScreenAdditionalFeedback.js

```diff
@@ -62,37 +74,42 @@ const style = css`
   }
 `;
 
-class ScreenInput extends React.Component<Props, State> {
+class ScreenAdditionalFeedback extends React.Component<Props, State> {
   state = {
     comment: '',
-    error: false,
   };
+
   handleChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
-    this.setState({ comment: e.target.value, error: false });
+    this.setState({ comment: e.target.value });
   };
+
   handleSubmit = (e: SyntheticEvent<HTMLFormElement>) => {
     e.preventDefault();
-    const { changeScreen, articleId } = this.props;
+    const { changeScreen, articleId, commentType } = this.props;
     const { comment } = this.state;
-    if (!comment.length) {
-      this.setState({
-        error: true,
-      });
-      return;
-    }
+
     createComment(
       articleId,
-      this.state.comment,
+      commentType,
+      comment,
       () => changeScreen(screensList.THANK_YOU),
       () => changeScreen(screensList.ERROR),
     );
+
+    simpleTracker('smartFAQCategories', {
```

Comment:
please update the readme (https://github.com/kiwicom/smart-faq#eventname-smartfaqcategories) with this new tracker and remove `letUsKnow`.